app/controllers/cases_controller.rb

```diff
@@ -21,7 +23,15 @@ class CasesController < ApplicationController
                   :process_date_responded,
                   :remove_pit_extension,
                   :record_late_team,
-                  :update_closure
+                  :update_closure,
+                  :response_upload_and_approve,
+                  :response_upload_for_redraft,
+                  :upload_responses,
+                  :upload_responses_action,
+                  :upload_response_and_approve,
+                  :execute_upload_response_and_approve,
```

Comment:
`:execute_upload_response_and_approve` should be further up the list to maintain the alphabetical ordering